packages/ui/src/components/tabs.tsx

```diff
@@ -81,7 +81,8 @@ export function Tabs({
   const valuesRef = useRef(values);
   valuesRef.current = values;
 
-  const valueToIdMapRef = useRef(new Map<string, string>());
+  const valueToIdMap = useMemo(() => new Map<string, string>(), []);
+  const valueToIdMapRef = useRef(valueToIdMap);
```

Comment:
I meant to use `useMemo` for the value prop passed to context provider

Reply:
ah... is 6b30b0fe what you meant?

I've also added the docs just now. Will also look into the changeset stuff (haven't used it before).

Reply:
I will add change set for you